packages/iov-tendermint-rpc/src/client.ts

```diff
@@ -1,7 +1,9 @@
+import { Stream } from "xstream";
+
 import { Adaptor, Decoder, Encoder, findAdaptor, Params, Responses } from "./adaptor";
-import { default as requests, Method } from "./requests";
+import { default as requests, Method, SubscribeRequestQuery, SubscriptionEventType } from "./requests";
```

Comment:
There were a LOT of things in requests. So rather than have a huge import here, I just used `requests.*` everywhere. Only exception was `Method`, which was used so much.

I would prefer for consistency to either refer to `requests.SubscribeRequestQuery` and `requests.SubscriptionEventType` and remove from the import.
OR
Explicitly import all the types from `requests.*` and drop the default import here

Reply:
Hmm, did not think about that. This was added by the IDE. In general, I'd prefer to just list them all like we do in the other places as long as the symbols are unique enough to not have conflicts. But for now, I go with the existing style

Reply:
But this import makes no sense, because there is no default export in `./requests`. This makes `requests.SubscriptionEventType` a type (maybe a TS bug) and `requests.SubscriptionEventType.NewBlock` unavailable, because `requests` is undefined at runtime.

I don't think there is syntax for importing everything from one module plus some separate symbols. You either have

```js
import * as requests from "./requests";
```

or

```js
import { all, the, things, Method } from "./requests";
```

Reply:
The name `Request` from "./requests" caused conflicts with some DOM type. Thus I go with `import * as requests from "./requests";`